src/main.rs

```diff
@@ -880,8 +881,26 @@ fn main() {
             path.map(|s| std::fs::remove_file(s).ok());
             0
         }
-        Err(e) => {
-            eprintln!("{e}");
+        Err(top_error) => {
+            eprint!("Error: ");
+            if top_error.source().is_none() {
+                eprintln!("Cloud Hypervisor exited with the following error:");
+                eprintln!("  {top_error}");
+            } else {
+                eprintln!("Cloud Hypervisor exited with the following chain of errors:");
+                eprintln!("  0: {top_error}");
+                let mut level = 1;
+                let mut next_error: &dyn StdError = &top_error;
+                while let Some(sub_error) = next_error.source() {
+                    next_error = sub_error;
+                    eprintln!("  {level}: {next_error}",);
+                    level += 1;
+                }
+            }
+
+            eprintln!();
+            eprintln!("Debug Info: {top_error:#?}");
```

Comment:
I think the commit message example doesn't match the current format with the extra `Error:` ( :+1: ) and the debug info now always being shown.

Reply:
Yes, thanks. It is still WIP and the prerequisites are not merged all so far. Will change this soon!